tiatoolbox/tools/registration/prealignment.py

```diff
@@ -0,0 +1,111 @@
+import cv2
+import numpy as np
+import scipy.ndimage as ndi
+from skimage import exposure
+from skimage.util import img_as_float
+
+from tiatoolbox.utils.metrics import dice
+
+
+def prealignment(
+    fixed_img, moving_img, fixed_mask, moving_mask, dice_overlap=0.5, rotation_step=10
+):
```

Comment:
I am getting flake8 error for this function definition. This is after I added the dice_overlap argument. Any suggestion?

Reply:
Here is an example to fix this error https://github.com/TissueImageAnalytics/tiatoolbox/pull/379/commits/f443ff148d0bceca2c117c4984cbab435d9bddd3